build.gradle.kts

```diff
@@ -306,6 +306,10 @@ dependencies {
     testRuntimeOnly(alchemist("cognitive-agents"))
     testRuntimeOnly(alchemist("physical-agents"))
 
+    implementation("it.unibo.alchemist:alchemist-incarnation-protelis:_")
+    implementation("it.unibo.alchemist:alchemist-incarnation-sapere:_")
+    implementation("it.unibo.alchemist:alchemist-swingui:_")
```

Comment:
This one smells a lot, and it is actually not acceptable: why do you need compile-time dependencies on these subprojects?